src/presets/firebase/preset.ts

```diff
@@ -68,4 +77,37 @@ const firebase = defineNitroPreset(
   }
 );
 
-export default [firebase] as const;
+const firebaseAppHosting = defineNitroPreset(
+  {
+    extends: "node-server",
+    serveStatic: true,
+    hooks: {
+      async compiled(nitro) {
+        await writeFile(
+          join(nitro.options.rootDir, ".apphosting/bundle.yaml"),
```

Comment:
@Yuangwang Important note here: We write it to project root (instead of monorepo root)

Reply:
That sounds good to me for now, thanks for the note